docs/current/learn-substrate/deployment/account-prefix.md

```diff
@@ -0,0 +1,15 @@
+---
+slug: account-prefix
+lang: en
+title: Account Prefix
+---
+
+* What is an Account Prefix?
+
+* What will it change?
+
+* Will it change the underlying private key/seed?
+
+* Can I use one key across multiple networks?
+
+* What are the current set of prefixes?
```

Comment:
Do we really want to copy this stuff over from the old site? IMO this should be an issue titled something like "Add docs for account prefix" and the bullets could go in the description.